youtube_dl/extractor/instagram.py

```diff
@@ -173,6 +188,27 @@ def _real_extract(self, url):
                 media = try_get(
                     additional_data, lambda x: x['graphql']['shortcode_media'],
                     dict)
+                if not media:
+                    # In some cases, the response from the server has items in it
+                    # Here is a sample of code that works fine with one item, but
+                    # I don't know for multiple items, cuz I don't have test links
+                    # for those.
+                    items0 = try_get(additional_data, lambda x: x['items'][0], dict)
+                    if items0:
+                        best_quality = items0.get("video_versions")[0]
+                        video_url = best_quality.get('url')
+                        height = int_or_none(best_quality.get('height'))
+                        width = int_or_none(best_quality.get('width'))
+                        description = items0.get('caption').get('text')
```

Comment:
```suggestion
                        description = try_get(additional_data, lambda x: x['caption']['text'], dict)
```
otherwise this would error out when no caption object